node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,9 +137,19 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType ? widgetData.previews.targetType.split(';')[0] : false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes && navigator.mimeTypes[type] && navigator.mimeTypes[type].enabledPlugin;
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
-                oae.api.util.template().render($('#linkpreview-iframe-template', $rootel), {'link': link}, $('#linkpreview-container'), $rootel);
+            if (embeddable && supported) {
+                if (type.substring(0, 6) === 'image/') {
+                    oae.api.util.template().render($('#linkpreview-image-template', $rootel), {'img': link, 'linkProfile': widgetData}, $('#linkpreview-container'), $rootel);
```

Comment:
Hm, I can no longer get this to work now. Every time I add the link `http://www.sanfrancisco.travel/sites/sftraveldev.prod.acquia-sites.com/files/SanFrancisco_0.jpg`, I get the following error:

```
    }
[2015-03-11T05:37:43.639Z] ERROR: oae-preview-processor/5563 on Nicolaass-MacBook-Pro.local: Could not generate a screenshot.
    Error: PhantomJS exited with return value 1
        at ChildProcess.<anonymous> (/Users/Nicolaas/Desktop/ApereoOAE/Code/oae/Hilary/node_modules/webshot/lib/webshot.js:222:61)
        at ChildProcess.EventEmitter.emit (events.js:98:17)
        at Process.ChildProcess._handle.onexit (child_process.js:789:12)
[2015-03-11T05:37:43.640Z] ERROR: oae-preview-processor/5563 on Nicolaass-MacBook-Pro.local: Could not generate an image (contentId=c:cam:Xyv0hbzK)
    err: {
      "code": 500,
      "msg": "PhantomJS exited with return value 1"
    }
```

Is anyone else able to reproduce this /cc @mrvisser @simong